modules/ROOT/pages/email/email-list.adoc

```diff
@@ -174,4 +173,17 @@ The IMAP matcher looks like this:
   answered="EXCLUDE|INCLUDE|REQUIRE"/>
 ----
 
-Notice that the IMAP matcher includes the `recent`, `seen`, `deleted`, and `answered` parameters.
\ No newline at end of file
+Notice that the IMAP matcher includes the `recent`, `seen`, `deleted`, and `answered` parameters.
+
+== Naming Strategy For Attachments
+
+Email Connector provides a parameter to specify a strategy for how to name attachments for the List and On New Email operations. This parameter is set at the configuration level, but can be overridden at the operation
```

Comment:
"provides a parameter to specify a strategy for how to name attachments" makes my head hurt, although it's not wrong. Oh. This is the thing from your Slack message, right? What I said there.